base/linalg/bunchkaufman.jl

```diff
@@ -88,8 +88,15 @@ size(B::BunchKaufman, d::Integer) = size(B.LD, d)
 issymmetric(B::BunchKaufman) = B.symmetric
 ishermitian(B::BunchKaufman) = !B.symmetric
 
+issuccess(B::BunchKaufman) = B.info == 0
+
+function Base.show(io::IO, F::BunchKaufman)
+    println(io, summary(F))
```

Comment:
Oops. Too fast. The problem here is that constructing the factor is a mess and not that useful.